va/va.h

```diff
@@ -1878,7 +1878,9 @@ typedef enum
     /** \brief Buffer type used for FEI input frame level parameters */
     VAEncMiscParameterTypeFEIFrameControl = 18,
     /** \brief encode extension buffer, ect. MPEG2 Sequence extenstion data */
-    VAEncMiscParameterTypeExtensionData = 19
+    VAEncMiscParameterTypeExtensionData = 19,
+    /** \brief adjust context parameters dynamically, refer \c VAEncMiscParameterContextParameterAdjustment */
+    VAEncMiscParameterTypeContextParameterAdjust = 20
```

Comment:
The context priority is a general parameter for decoder, encoder, vpp.  The value name started with "VAEncMiscParameter" will cause the confusion.

Reply:
accept